### DISCUSSION TITLE
Meshing

### DISCUSSION THREAD

**Discussion by ap264** [neperfepx/neper#566] [Q&A] [answered]
Hi romain

I would like to know if it is possible to generate my Voronoi model and mesh it exclusively with 3d elements. I mean if I mesh my model with default configurations, I would generate a mesh with some 2d elements and some with 3d elements. But I would like to know if I can generate without any 2d elements. 

Thank you

**Comment by rquey** [comment, marked as answer]
Use `-dim 3` (or `-dim3:msh` for msh output), see  [-dim](https://neper.info/doc/neper_m.html#cmdoption-dim).

**Reply by ap264** [reply]
thank you, sir. I will keep you posted if this does not work.
